doc/source/parameters/cfd/volume_of_fluid.rst

```diff
@@ -0,0 +1,203 @@
+  
+  .. tip::
+    This parameter must be larger than 1 for interface sharpening. Choosing values less than 1 leads to interface smoothing instead of sharpening. A good value would be between 1 and 2.
+  
+  * ``sharpening frequency``: sets the frequency (in number of iterations) for the interface sharpening computation.
+
+.. seealso::
+
+  The :doc:`../../examples/multiphysics/dam-break-VOF/dam-break-VOF` example using VOF represents well the interface sharpening issue.
+
+* ``subsection peeling wetting``: defines the parameters for peeling and wetting mechanisms at the VOF boundaries, as given in :doc:`boundary_conditions_multiphysics`.
+
+  * ``enable``: controls if peeling/wetting mechanism is enabled.
+  * ``verbosity``: enables the display of the number of peeled and wet cells at each time-step. Choices are: ``quiet`` (default, no output) and ``verbose``.
+
+  .. admonition:: Example of a ``set verbosity = verbose`` output:
+  
+    .. code-block:: text
+
+      Peeling/wetting correction at step 2
+        -number of wet cells: 24
+        -number of peeled cells: 1
+
+  * Peeling of the higher density fluid occurs where those conditions are met:
+
+    * the cell is in the domain of the higher density fluid,
+    * the cell pressure value is below ``peeling pressure value``, and
+    * the more than half of the quadrature points in the cell have a pressure gradient below ``peeling pressure gradient``.
+
+    The cell is then filled with the lower density fluid by changing its phase value.
+
+  * Wetting of the lower density fluid occurs where those conditions are met: 
+
+    * the cell is in the domain of the lower density fluid,
+    * the cell pressure value is above ``wetting pressure value``, and
+    * the distance (on the phase value) to the interface is above ``wetting phase distance``.
+
+    The cell is then filled with the higher density fluid by changing its phase value.
+
+    .. tip::
+
+      For ``set wetting phase distance = 0``, the wetting can only occur at the interface (considered at ``phase value = 0.5``).
+
+      For ``set wetting phase distance`` :math:`> 0`, the wetting can occur in the area where is larger than the area occupied by the higher density fluid. For example:
+
+      * if the ``fluid 1`` has a higher density than ``fluid 0``, and ``set wetting phase distance = 0.1``, the wetting can occur where the phase value is below :math:`= 0.4`.
+      * if the ``fluid 0`` has a higher density than ``fluid 1``, and ``set wetting phase distance = 0.1``, the wetting can occur where the phase value is above :math:`= 0.6`.
+
+* ``diffusion``: value of the diffusion in the transport equation of the phase fraction. Default value is 0 to have pure advection. This can be used to :ref:`improve wetting`.
```

Comment:
right now `diffusion` is outside of the peeling wetting subsection. Though I think I will put it inside of it, because I do not see any other case where we would like to add a diffusivity if not to improve the wetting mechanism! ^^